docs/reference/search/search-your-data/search-across-clusters.asciidoc

```diff
@@ -610,12 +670,339 @@ were searched across all clusters and that all were successful.
 <3> The `_clusters` section shows that searches on all 3 clusters were successful.
 
 
+[discrete]
+[[cross-cluster-search-failures]]
+=== {ccs-cap} failures
+
+Failures during a {ccs} can result in one of two conditions:
+
+. partial results (2xx HTTP status code)
+. a failed search (4xx or 5xx HTTP status code)
+
+Failure details will be present in the search response in both cases.
+
+A search will be failed if a cluster marked with `skip_unavailable`=`false`
+is unavailable, disconnects during the search, or has search failures on
+all shards. In all other cases, failures will result in partial results.
+
+Search failures on individual shards will be present in both the `_shards`
+section and the `_clusters` section of the response.
+
+A failed search will have an additional top-level `errors` entry in the response.
+
+Here is an example of a cluster with partial results due to search failure
+a one shard. The query would be similar to ones shown previously. We
```

Comment:
"on one shard" instead of "a one shard"?

Reply:
Fixed.